doc/snek.adoc

```diff
@@ -1185,7 +1226,7 @@ value for _bound_ is ``–1``. Here are some examples:
 
 String interpolation in Snek can be confused with formatted printing
 in other languages. In Snek, the ``print`` function takes a single
```

Comment:
Hrm. the Snek print function takes as many values as you like. And fails to separate them with spaces. Ok, fixed that bug. Now I've got this for text:

    Print writes all of its positional parameters to the console separated
    by spaces (`' '`) followed by the `end` named parameter (default:
    `'\n'`).(((print)))

Reply:
Hrm. 'to the console' is kinda buried in there. Where should it go?

Reply:
I think it's fine as you wrote it.